File: src/test/java/com/sat/documentation/LoginController.java
package com.sat.documentation;

import com.sat.auth.application.dto.TokenPair;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class LoginController extends Documentation {

    @PostMapping("/token")
    public ResponseEntity<TokenPair> login(@RequestBody TokenRequest request) {
        String accessToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzd...";
        String refreshToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzd...";
        return ResponseEntity.ok(new TokenPair(accessToken, refreshToken));
    }

    public record TokenRequest(
            String accessToken
    ) {

    }


    public record RefreshTokenRequest(
            String type,
            String refreshToken
    ) {

    }
}

File: src/main/kotlin/com/sat/user/ui/web/LoginController.kt
package com.sat.user.ui.web

import com.sat.user.application.command.dto.LoginCommand
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RequestBody
import org.springframework.web.bind.annotation.RestController

@RestController
class LoginController {

    @PostMapping("/login")
    fun login(@RequestBody command: LoginCommand) {
        TODO("JWT 응답")
    }
}
